Tools/AP_Bootloader/board_types.txt

```diff
@@ -314,6 +314,15 @@ AP_HW_AIRVOLUTE_DCS2                 5200
 AP_HW_AOCODA-RC-H743DUAL             5210
 AP_HW_AOCODA-RC-F405V3               5211
 
+# IDs 5220-5239 reserved for UAV-DEV GmbH
+AP_HW_UAV-DEV-HAT-H7                 5220
+AP_HW_UAV-DEV-NucPilot-H7            5221
+AP_HW_UAV-DEV-M10S-L4                5222
+AP_HW_UAV-DEV-F9P-G4                 5523
+AP_HW_UAV-DEV-UM982-G4               5524
+AP_HW_UAV-DEV-M20D-G4                5525
+AP_HW_UAV-DEV-Sensorboard-G4         5526
```

Comment:
These ones are out-of-order numerically.  Did you mean to have all of these consecutive?